testplan/testing/multitest/base.py

```diff
@@ -1,72 +1,68 @@
-"""Multitest main test execution framework."""
+"""MultiTest test execution framework."""
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```

Comment:
I get OCD with this. My OCD is satiated with this:

```py
from __future__ import (
    unicode_literals, print_function, division, absolute_import
)
```

Reply:
No change needed. I'll just manage my OCD better.

Reply:
It's funny, I get OCD when multiple imports are grouped on one line - though I totally get it's just a personal preference.

These lines were added by the `pasteurize` tool (https://python-future.org/quickstart.html#to-convert-existing-python-3-code), I just left them as-is.